src/Asset/Asset.spec.js

```diff
@@ -128,7 +129,7 @@ describe('Asset', function() {
             price: 40,
             buyer: null,
             status: Publication.STATUS.open,
-            type: Publication.TYPES.parcel,
+            type: ASSET_TYPE.parcel,
```

Comment:
We had a conversation with @NicoSantangelo about it. One of the key concepts of what we don't have `Parcel.type` or `Entity.type` is because we could have dynamic entities in the future creating by an admin and it would be hard to maintain something like `DynamicEntity.TYPE`